docs/api/en/objects/InstancedMesh.html

```diff
@@ -45,7 +45,10 @@ <h2>Properties</h2>
 		<h3>[property:Box3 boundingBox]</h3>
 		<p>
 			This bounding box encloses all instances of the [name]. Can be calculated
-			with [page:.computeBoundingBox](). Default is `null`.
+			with [page:.computeBoundingBox](). By default, this bounding box is used for
+			the purposes of frustum culling. You have to manually update the bounding box
+			if you transform any instances outside of the current bounds. Default is
```

Comment:
>  You have to manually update the bounding box if you transform any instances outside of the current bounds.

The bounding volumes should be updated whenever instances are transformed. Depending on how you transform instances, the bounds can be smaller than before and the bounding volumes should reflect that.